CMakeLists.txt

```diff
@@ -149,6 +149,9 @@ set(ABSL_LOCAL_GOOGLETEST_DIR "/usr/src/googletest" CACHE PATH
   "If ABSL_USE_GOOGLETEST_HEAD is OFF and ABSL_GOOGLETEST_URL is not set, specifies the directory of a local GoogleTest checkout."
   )
 
+option(ABSL_USE_HARDENED_MODE
+  "If ON, abseil will sue hardened mode." OFF)
```

Comment:
The work "use" is still misspelled here. Also, please capitalize Abseil.

Reply:
Fixed in commit 3e2bf646

Reply:
Reverted testing changes in commit 3082c45b

Reply:
I understand that double the number of builds and tests is generally not a good idea.

Thanks for pointing out bazel builds I found that the alternate options file is used there form environment so hardened mode is tested.

I don't know how you want to be the cmake option tested.
Normally I would create another shell script that:
    - check that default values for boolean options from 'cmake -L'
    - configure abseil with non-default boolean values
    - check that generated options.h match configured non-default values
but this I'm not sure I can develop it just in few code review cycles to match Abseil expectations.

I really don't know how to proceed further so please suggest how the test should look like and where should be placed etc.